topdown/cache/cache.go

```diff
@@ -110,11 +111,20 @@ func (c *cache) Delete(k ast.Value) {
 	c.unsafeDelete(k)
 }
 
+func (c *cache) UpdateConfig(config *Config) {
+	if config == nil {
```

Comment:
I wonder if we will hit this condition 🤔

Reply:
Not as it's currently coded, but I think it's prudent defensive code. It's hard to predict how folks wrapping OPA might use the cache.